docs/maintainer-guide/issues.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Managing Issues
+layout: doc
+---
+
+# Managing Issues
+
+New issues are filed frequently, and how we respond to those issues directly affects the success of the project. Being part of the project team means helping to triage and address issues as they come in so the project can continue to run smoothly.
+
+## Things to Keep in Mind
+
+1. **Be nice.** Even if the people are being rude or aggressive on an issue, as a project team member you must be the mature one in the conversation. Do your best to work with everyone no matter their style. Remember, poor wording choice can also be a sign of someone who doesn't know English very well, so be sure to consider that when trying to determine the tone of someone's message. Being rude, even when someone is being rude to you, reflects poorly on the team and the project as a whole.
+1. **Be inquisitive.** Ask questions on the issue whenever something isn't clear. Don't assume you understand what's being reported if there are details missing. Whenever you are unsure, it's best to ask for more information.
+1. **Not all requests are equal.** It's unlikely we'll be able to accommodate every request, so don't be afraid to say that something doesn't fit into the scope of the project or isn't practical. It's better to give such feedback if that's the case.
+1. **Close when appropriate.** Don't be afraid to close issues that you don't think will be done, or when it's become clear from the conversation that there's no further work to do. Issues can always be reopened if they are closed incorrectly, so feel free to close issues when appropriate. Just be sure to leave a comment explaining why the issue is being closed (if not closed by a commit).
+
+## When an Issue is Opened
+
+When an issue is first opened, follow these steps:
+
+1. Is it clear what is being requested? If no, ask for more information. If yes, then:
+    * Label bug reports with the "bug" label (also use the "accepted" label if you can reproduce and verify the bug)
+    * Label requests for changes to existing features with the "enhancement" and "proposal" labels
+    * Label requests for completely new features with the "feature" and "proposal" labels
+    * Use an appropriate label for the part of the project the issue refers to
+1. If it's a bug, be sure the issue includes the following (ask for any missing information):
```

Comment:
While I understand why you are asking for it, I don't think it's always necessary, if you can reproduce a bug with the information provided, it's redundant to ask for that information from the user. I think it can be very frustrating to new users to provide a lot of information, just to be asked to provide even more. I think this should be filed under "use your own judgement".